drivers/hmis/spec/requests/hmis/update_referral_posting_spec.rb

```diff
@@ -17,7 +19,7 @@
   let!(:link_creds) do
     create(:ac_hmis_link_credential)
   end
-  let!(:coc) { create :hud_project_coc, data_source: ds1, project_id: destination_project.project_id, state: 'MA' }
+  let!(:coc) { create :hud_project_coc, data_source: ds1, project_id: destination_project.project_id, state: 'MA', coc_code: 'MA-500' }
```

Comment:
I tested this without the diff and it doesn't seem like it's needed for the test to pass, am I missing something?